examples/src/common/camera.ts

```diff
@@ -1,8 +1,44 @@
-import type { box2D, vec2 } from '@alleninstitute/vis-geometry';
+import { Box2D, Vec2, type box2D, type vec2 } from '@alleninstitute/vis-geometry';
 
 // a basic camera, for viewing slices
 export type Camera = {
     readonly view: box2D; // a view in 'data space'
     readonly screen: vec2; // what that view projects to in display space, aka pixels
     readonly projection: 'webImage' | 'cartesian';
 };
+/**
+ * Zooms relative to your current mouse position
+ * @param view your current view
+ * @param screenSize the size of your canvas/screen
+ * @param zoomScale the scale you want to apply to your view
+ * @param mousePos the offsetX and offsetY of your mouse
```

Comment:
none of these comments are helpful are they?

a parameter named screenSize is obviously the size of the screen - maybe give the units?
mousePos - is this in pixels, or the data space? 
view - what is this? obviously its the current view. this is a box, in dataspace, that gets mapped to the screen or canvas.